jenkins/automation/vra_infra_provisioner.groovy

```diff
@@ -0,0 +1,95 @@
+pipeline {
+    agent {
+        node {
+            label 'terraform-node'
+        }
+    }
+    
+    options {
+        timeout(time: 30, unit: 'MINUTES')
+        timestamps()
+        buildDiscarder(logRotator(daysToKeepStr: '30', numToKeepStr: '30'))
+        ansiColor('xterm')
+    }
+
+
+    parameters {
+        string(name: 'CORTX_RE_REPO', defaultValue: 'https://github.com/Seagate/cortx-re', description: 'Repository for VRA infra provisioner script', trim: true)
+        string(name: 'CORTX_RE_BRANCH', defaultValue: 'main', description: 'Branch or GitHash for VRA infra provisioner script', trim: true)
+        string(name: 'VRA_USERNAME', defaultValue: '', description: 'User ID used to login VRA console', trim: true)
+        password(name: 'VRA_PASSWORD', description: 'Password used to login VRA console')
+        string(name: 'VM_NAMES', defaultValue: '', description: 'list of VM names need to be procured. (comma separated list of VM names)', trim: true)
+        choice(
+            name: 'VM_CPU',
+            choices: ['4', '2', '6', '8'],
+            description: 'Number of vCPU required for VM'
+        )
+        choice(
+            name: 'VM_MEMORY',
+            choices: ['4096', '2048', '8192', '16384'],
```

Comment:
You can keep choices as 2GB,4GB etc.

Reply:
@pankajgborole, `2GB` is not recognized by terraform code, so kept `2048` so variable could directly used by code